app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -35,25 +39,35 @@
 
     private int position;
     private Contribution contribution;
+    Random random =new Random();
```

Comment:
default visibility

Reply:
not spaced to codestyle either

Reply:
Done, stupid of me to miss these

Reply:
happens to the best of us, `ctral+alt+shift+l` will expose the reformat dialog so you can select vcs changed text, don't think it'll work now tho